src/holdingpen/containers/DashboardPage.jsx

```diff
@@ -1,10 +1,46 @@
 import React, { Component } from 'react';
+import { Row, Col } from 'antd';
+import data from '../report-errors.json';
+import CategoryCard from '../components/CategoryCard';
+import ExceptionsTable from '../components/ExceptionsTable';
 
 class DashboardPage extends Component {
+  static calculateExceptionsMetadata() {
+    const exceptions = [...data];
+    return exceptions.reduce((acc, exception) => {
+      const tally = { ...acc };
```

Comment:
I was working on `acc` instead of it's copy `tally`, but eslint was showing the following error:
`[eslint] Assignment to property of function parameter 'tally'. (no-param-reassign)`